.travis.yml

```diff
@@ -60,15 +60,8 @@ jobs:
   fast_finish: true
   # These need to be exact matches, including whitespace!
   allow_failures:
-    # PHP unit tests (7.3, WordPress trunk)
-    - env: WP_VERSION=trunk  DEV_LIB_ONLY=phpunit                               INSTALL_PWA_PLUGIN=1
     # PHP and JavaScript unit tests (7.3, WordPress trunk, with code coverage)
     - env: WP_VERSION=latest DEV_LIB_ONLY=phpunit                               INSTALL_PWA_PLUGIN=1 RUN_PHPUNIT_COVERAGE=1
```

Comment:
@swissspidy  Should this still be allowed to fail? Why was added initially?

Reply:
Because trunk is not guaranteed to be stable. #4029 just showed it this week :-)

See #2408 for our discussion about adding this test & marking it as allowed to fail.